modules/virt-configuring-configmap-for-obsolete-cpu-models.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/node_maintenance/virt-managing-node-labeling-obsolete-cpu-models.adoc
+
+[id="virt-configuring-configmap-for-obsolete-cpu-models_{context}"]
+= Configuring a ConfigMap for obsolete CPU models
+
+Use this procedure to configure a ConfigMap for obsolete CPU models.
+
+.Procedure
+
+* Create a ConfigMap object, specifying the obsolete CPU models in the `obsoleteCPUs` array. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: <cpu-plugin-configmap> <1>
```

Comment:
Please put the name of the configmap in here.
It needs to be a very specific name. The user can not choose it.

Reply:
Removed the brackets to indicate that this is the exact configmap name specified in the file.